frontend/src/AppRoot.tsx

```diff
@@ -61,53 +63,37 @@ const AppRoot = () => {
           <Route path={ROUTES.HOME} exact>
             {isAuthenticated ? <HomePage /> : <PublicHomePage />}
           </Route>
-          <PrivateRoute
-            path={ROUTES.ADMIN_ROOT}
-            isAuthenticated={isAuthenticated}
-            exact
-          >
+          <PrivateRoute path={ROUTES.ADMIN_ROOT} exact>
```

Comment:
A further improvement would be to map an array of routes and components, but we need to be mindful of the order of routes. For example, `route/new` must go BEFORE `route/:id`.